tools/appveyor.psm1

```diff
@@ -29,6 +94,21 @@ function Invoke-AppVeyorBuild
 # Implements the AppVeyor 'install' step
 function Invoke-AppVeyorInstall
 {
+    if(Test-DailyBuild){
+        $buildName = "[Daily]"
+        if($env:APPVEYOR_PULL_REQUEST_TITLE)
+        {
+            $buildName += $env:APPVEYOR_PULL_REQUEST_TITLE
+        }
+        else
+        {
+            $buildName += $env:APPVEYOR_REPO_COMMIT_MESSAGE
+        }
+
+        Update-AppveyorBuild -message $buildName
+    }
+
+    Set-BuildVariable -Name TestPassed -Value True
```

Comment:
resolved

---
In reply to: [77055496](https://github.com/PowerShell/PowerShell/pull/2139#discussion_r77055496) [](ancestors = 77055496)